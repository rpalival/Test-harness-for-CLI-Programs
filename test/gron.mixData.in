{"str": "hello", "num": 123, "bool": true, "null": null}