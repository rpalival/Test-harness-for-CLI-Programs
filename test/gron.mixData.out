json = {};
json.str = "hello";
json.num = 123;
json.bool = true;
json.null = null;